sdk/python/feast/infra/offline_stores/file_source.py

```diff
@@ -114,7 +121,6 @@ def from_proto(data_source: DataSourceProto):
             path=data_source.file_options.uri,
             event_timestamp_column=data_source.event_timestamp_column,
             created_timestamp_column=data_source.created_timestamp_column,
-            date_partition_column=data_source.date_partition_column,
```

Comment:
I think you can make this same change for all the other sources

Reply:
I *think* I have? Assuming you're referring to not passing `date_partition_column` into the constructor here?

Reply:
ah I meant remove from `from_proto` and `to_proto`, e.g. I don't think you did that for `bigquery_source.py`